.gitignore

```diff
@@ -1,2 +1,3 @@
 .pytest_cache
 __pycache__/
+.vscode
```

Comment:
can you add a newline after this https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline